tests/MongoDB.Driver.Tests/Specifications/Runner/DisposableJsonDrivenTestRunner.cs

```diff
@@ -0,0 +1,43 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+
+namespace MongoDB.Driver.Tests.Specifications.Runner
+{
+    public abstract class DisposableJsonDrivenTestRunner : MongoClientJsonDrivenTestRunnerBase, IDisposable
+    {
+        protected List<IDisposable> Disposables { get; } = new List<IDisposable>();
```

Comment:
I'd rather keep the list private and add an `AddDisposable` helper method:

    protected void AddDisposable(IDisposable disposable)
    {
        _disposables.Add(disposable);
    }